crates/engine/tree/src/tree/state.rs

```diff
@@ -1,7 +1,9 @@
 //! Functionality related to tree state.
 
 use crate::engine::EngineApiKind;
-use alloy_eips::{eip1898::BlockWithParent, BlockNumHash};
+#[cfg(test)]
+use alloy_eips::eip1898::BlockWithParent;
```

Comment:
instead of feature gating this, could we use the full path where we currently use `BlockWithParent` then we can remove this import here